src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -48,19 +46,18 @@
 @RunWith(MockitoJUnitRunner.StrictStubs.class)
 @Category(Unit.class)
 public class SearchServiceTest {
-  private static final String sqlQuery =
-      "SELECT GENERATE_UUID() uuid, CURRENT_TIMESTAMP() AS example_now"
-          + " FROM UNNEST(GENERATE_ARRAY(1, 3));";
-
   private static final String timPropertyName = "example:identifier.now";
   private static final String timEncodedName = TimUtils.encode(timPropertyName);
 
+  private static final String sqlQuery =
+      String.format(
+          "SELECT GENERATE_UUID() uuid, CURRENT_TIMESTAMP() AS [%s]"
+              + " FROM UNNEST(GENERATE_ARRAY(1, 3));",
```

Comment:
I think it would be better to have a format string that works both here and below in `timColumnEncodingTest()`, e.g. in this case the code could add the `[]`s around the name before using `String.format()`.